src/zenml/cli/service_connectors.py

```diff
@@ -992,16 +1010,48 @@ def list_service_connectors(
         )
 
     connectors = client.list_service_connectors(**kwargs)
-    if not connectors:
+    if not connectors.items:
         cli_utils.declare("No service connectors found for the given filters.")
         return
 
-    cli_utils.print_service_connectors_table(
-        client=client,
-        connectors=connectors.items,
-        show_active=not is_sorted_or_filtered(ctx),
+    show_active = not is_sorted_or_filtered(ctx)
```

Comment:
Just noticed something. When you list items that can be `active`, the column that shows the state with either `●` or `-` does not have a header name. Same goes for other element like stacks or users.